test/new-e2e/pkg/runner/local_profile.go

```diff
@@ -43,7 +44,27 @@ func NewLocalProfile() (Profile, error) {
 	if err != nil {
 		return nil, err
 	}
-	return localProfile{baseProfile: newProfile("e2elocal", strings.Split(environments, " "), store, nil)}, nil
+	outputDir, err := getLocalOutputDir()
+	if err != nil {
+		return nil, err
+	}
+	return localProfile{baseProfile: newProfile("e2elocal", strings.Split(environments, " "), store, nil, outputDir)}, nil
+}
+
+func getLocalOutputDir() (string, error) {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Why do you need to walk to the `.git` folder on local tests ? What about using `home` like we do for the config file ? At the end of the day you want a common, writable directory, 	`home` or `temp` could serve that need. 

This code is cool, but I feel like it's too complex for the need.

Reply:
I wanted a single location across tests so the source tree wouldn't be cluttered with output files, so repo root seemed like a reasonable location and this seemed like a reasonable way to find it. I didn't think about using `home`, and that might make more sense since the `outputDir` parameter in the config will also be used across projects.